OWNERS_ALIASES

```diff
@@ -222,6 +222,7 @@ aliases:
     - justaugustus # SIG Chair
     - saschagrunert # Branch Manager
     - tpepper # SIG Chair / Patch Release Team
+    - xmudrii # Branch Manager
```

Comment:
@justaugustus Should we replace Branch Manager and Patch Release Team with Release Manager?

Reply:
Yes please! :)

Reply:
@justaugustus Done, PTAL.